jinahub/encoders/text/TransformerTorchEncoder/tests/data/test_data.txt

```diff
@@ -0,0 +1,50 @@
+The ROMEO website, iOS app and Android app are commonly used by the male gay community to find friends, dates, love or get informed about LGBT+ topics.
```

Comment:
Could you remove this file, since it seems it is not used anywhere in the tests? Or am I mistaken?

Reply:
it is needed in  `conftest.py` actually.